resources/assets/less/bem/beatmap-icon.less

```diff
@@ -46,45 +48,45 @@
 
   &--easy {
     &::before {
-      background-color: @green;
+      background-color: @diff-easy;
     }
   }
 
   &--normal {
     &::before {
-      background-color: @blue;
+      background-color: @diff-normal;
     }
   }
 
   &--hard {
     &::before {
-      background-color: @yellow;
+      background-color: @diff-hard;
     }
   }
 
   &--insane {
     &::before {
-      background-color: @pink;
+      background-color: @diff-insane;
     }
   }
 
   &--expert {
     &::before {
-      background-color: @purple;
+      background-color: @diff-expert;
     }
   }
 
   &--expert-plus {
     &::before {
-      background-color: #000;
+      background-color: @diff-expert-plus;
```

Comment:
I forgot to mention this should use a different color (white?) because it's currently unreadable.